package.json

```diff
@@ -81,12 +81,12 @@
     "onfido-sdk-ui": "10.3.0",
     "process": "^0.11.10",
     "prop-types": "^15.7.2",
-    "pusher-js": "^7.0.6",
-    "react": "18.1.0",
+    "pusher-js": "7.4.0",
```

Comment:
upgraded pusher because of this issue https://github.com/pusher/pusher-js/issues/654#issuecomment-1326676074